packages/frontend/src/App.tsx

```diff
@@ -30,6 +37,21 @@ const router = createBrowserRouter([
         index: true,
         element: <Landing />,
       },
+      {
+        path: "/staff/inventory/manage",
+        loader: profileLoader(queryClient),
```

Comment:
you can create a parent route that just uses the loader once. e.g. 
```tsx
{
  path: "/staff"
  loader: profileLoader(queryClient),
}
```
Child routes will get data from their nearest parent loader.